data/settings/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2023 The Android Open Source Project
+ * Copyright (C) 2023-2024 The Android Open Source Project
```

Comment:
may want to update the copyright years as per #115

Reply:
Done. Updated this file and the rest of the files I had edited.